spec.go

```diff
@@ -1,5 +1,9 @@
 package certificatetpr
 
+type Searcher interface {
+	SearchCerts(clusterID string) (AssetsBundle, error)
+}
```

Comment:
Here is the new interface packages can use to be able to mock out the certificate watcher. This happens for instance in the kvm-operator. See https://github.com/giantswarm/kvm-operator/pull/150.